internal/kafka/internal/cmd/kafka/list.go

```diff
@@ -48,6 +49,9 @@ func runList(env *environments.Env, cmd *cobra.Command, _ []string) {
 	var kafkaService services.KafkaService
 	env.MustResolveAll(&kafkaService)
 
+	var kafkaConfig config.KafkaConfig
+	env.MustResolveAll(&kafkaConfig)
```

Comment:
Was testing this and found that the kafkaConfig type needs to be a pointer otherwise it can't find it in the di container. 

Also found that `MustResolveAll` can take multiple args so we don't need to call it twice :) 

```suggestion
	var kafkaService services.KafkaService
	var kafkaConfig *config.KafkaConfig
	env.MustResolveAll(&kafkaService, &kafkaConfig)
```